Library/Formula/dbxml.rb

```diff
@@ -0,0 +1,27 @@
+require 'formula'
+
+class Dbxml < Formula
+  url 'http://download.oracle.com/berkeley-db/dbxml-6.0.17.tar.gz'
+  homepage 'http://www.oracle.com/us/products/database/berkeley-db/xml/overview/index.html'
+  sha1 'fa706a73497bbdc6e026cd83e8a5538dd9468e0c'
+
+  depends_on 'xerces-c'
+  depends_on 'xqilla'
+  depends_on 'berkeley-db'
+
+  def install
+    inreplace "dbxml/configure" do |s|
+      s.gsub! "lib/libdb-*.la | sed 's\/.*db-\\\(.*\\\).la", "lib/libdb-*.a | sed 's/.*db-\\(.*\\).a"
+      s.gsub! "lib/libdb-*.la", "lib/libdb-*.a"
+    end
```

Comment:
I did too but MySQL has a different system to manage its code. It has a public bug report system. For other oracle products except forums I don't think they have a similar system. If you have more information than me I would do it with pleasure